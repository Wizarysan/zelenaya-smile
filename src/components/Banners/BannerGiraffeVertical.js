import React from 'react';

//TODO выделить пути features в отдельный файл с константами

import giraffeWhole from './../../assets/images/banners/giraffe-whole.png';
import giraffeWeb from './../../assets/images/banners/giraffe-web.png';
import giraffeTv from './../../assets/images/banners/giraffe-tv.png';
import giraffeMediac from './../../assets/images/banners/giraffe-mediac.png';
import giraffePlay from './../../assets/images/banners/giraffe-play.png';
import './BannerGiraffe.scss';

//TODO выбор города для линка в баннере

const BannerGiraffeVertical = (props) => {
  return(
    <div className="giraffe-v">
      <div className="giraffe back-greygrad">
        <a href="http://stavropol.zelenaya.net/mixer/" >
          <div className="giraffe__wrapper">
            <div className="giraffe__title">
              <span>интернет и тв</span>
              <div className="giraffe__title-big">ОГРОМНЫХ</div>
              <span>возможностей</span>
            </div>
            <div className="giraffe__whole">
              <img src={giraffeWhole} className="giraffe__whole-img"/>
            </div>
            <div className="giraffe__options">
              <div className="giraffe__option">
                <img src={giraffeWeb} className="giraffe__option-img"/>
                <div>Интернет<br className="giraffe__option-br"/> тарифы до<br className="giraffe__option-br"/> 100 Мбит/с</div>
              </div>
              <div className="giraffe__option">
                <img src={giraffeTv} className="giraffe__option-img"/>
                <div>Цифровое ТВ<br className="giraffe__option-br"/> более 250<br className="giraffe__option-br"/> каналов</div>
              </div>
              <br className="giraffe__options-divider"/>
              <div className="giraffe__option">
                <img src={giraffePlay} className="giraffe__option-img"/>
                <div>Бесплатное<br className="giraffe__option-br"/> управление<br className="giraffe__option-br"/> ТВ-эфиром</div>
              </div>
              <div className="giraffe__option">
                <img src={giraffeMediac} className="giraffe__option-img"/>
                <div>Бесплатный<br className="giraffe__option-br"/> медиацентр <br className="giraffe__option-br"/>в комплекте</div>
              </div>
            </div>
          </div>
        </a>
      </div>
    </div>
  )
}

export default BannerGiraffeVertical;
