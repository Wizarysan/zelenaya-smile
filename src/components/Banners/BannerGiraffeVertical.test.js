import React from 'react';
import BannerGiraffeVertical from './BannerGiraffeVertical.js';
import renderer from 'react-test-renderer';

it('renders correctly with test city', () => {
  const tree = renderer
    .create(<BannerGiraffeVertical cityGW="testCity"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});