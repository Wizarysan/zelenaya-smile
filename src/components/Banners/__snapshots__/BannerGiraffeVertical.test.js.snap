exports[`test renders correctly with test city 1`] = `
<div
  className="giraffe-v">
  <div
    className="giraffe back-greygrad">
    <a
      href="http://testCity.zelenaya.net/mixer/">
      <div
        className="giraffe__wrapper">
        <div
          className="giraffe__title">
          <span>
            интернет и тв
          </span>
          <div
            className="giraffe__title-big">
            ОГРОМНЫХ
          </div>
          <span>
            возможностей
          </span>
        </div>
        <div
          className="giraffe__whole">
          <img
            className="giraffe__whole-img"
            src="giraffe-whole.png" />
        </div>
        <div
          className="giraffe__options">
          <div
            className="giraffe__option">
            <img
              className="giraffe__option-img"
              src="giraffe-web.png" />
            <div>
              Интернет
              <br
                className="giraffe__option-br" />
               тарифы до
              <br
                className="giraffe__option-br" />
               100 Мбит/с
            </div>
          </div>
          <div
            className="giraffe__option">
            <img
              className="giraffe__option-img"
              src="giraffe-tv.png" />
            <div>
              Цифровое ТВ
              <br
                className="giraffe__option-br" />
               более 250
              <br
                className="giraffe__option-br" />
               каналов
            </div>
          </div>
          <br
            className="giraffe__options-divider" />
          <div
            className="giraffe__option">
            <img
              className="giraffe__option-img"
              src="giraffe-play.png" />
            <div>
              Бесплатное
              <br
                className="giraffe__option-br" />
               управление
              <br
                className="giraffe__option-br" />
               ТВ-эфиром
            </div>
          </div>
          <div
            className="giraffe__option">
            <img
              className="giraffe__option-img"
              src="giraffe-mediac.png" />
            <div>
              Бесплатный
              <br
                className="giraffe__option-br" />
               медиацентр 
              <br
                className="giraffe__option-br" />
              в комплекте
            </div>
          </div>
        </div>
      </div>
    </a>
  </div>
</div>
`;
